mesonbuild/dependencies/configtool.py

```diff
@@ -170,4 +170,4 @@ def get_variable(self, *, cmake: T.Optional[str] = None, pkgconfig: T.Optional[s
                 return variable
         if default_value is not None:
             return default_value
-        raise DependencyException(f'Could not get config-tool variable and no default provided for {self!r}')
+        raise DependencyException(f'Could not get {self.name} config-tool variable {configtool} and no default provided')
```

Comment:
IIRC this is generic support for config tools that work like pkg-config but not quite. Does it make sense to report which config tool this is about here?

Reply:
Good point, added tool's name.